portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,28 +39,47 @@
 public class DataServlet extends HttpServlet {
 
   private static final String COMMENT_ENTITY_KEY = "Comment";
-  private static final String COMMENT_PROPERTY_NAME = "comment";
-  private static final String TIME_PROPERTY_NAME = "dateTime";
+  private static final String COMMENT_PROPERTY_NAME = "userComment";
+  private static final String TIME_PROPERTY_NAME = "dateTimeCreated";
   
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query(COMMENT_ENTITY_KEY)
+                  .addSort("dateTimeCreated", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Comments> comments = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String text = (String) entity.getProperty(COMMENT_PROPERTY_NAME);
+      Date dateTimeCreated = (Date) entity.getProperty(TIME_PROPERTY_NAME);
+
+      Comments userComment = new Comments(id, text, dateTimeCreated);
+      comments.add(userComment);
+    }
+
+    Gson gson = new Gson();
+    response.setContentType("application/json;");
+    response.getWriter().println(gson.toJson(comments));
+  }
+
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String text = request.getParameter(COMMENT_PROPERTY_NAME);
-    long time = System.currentTimeMillis();
+    String userComment = request.getParameter("comment");
```

Comment:
👍 : I agree that your request param names are and should be distinct from your data store's property names.
👎 : It's still better to reference a constant than a magic string.

Maybe `COMMENT_PARAM_NAME`?